descartes_core/CMakeLists.txt

```diff
@@ -3,13 +3,12 @@ project(descartes_core)
 
 add_definitions(-std=gnu++0x)
 find_package(catkin REQUIRED COMPONENTS
-  console_bridge
   moveit_core
   roscpp
+  cmake_modules
 )
 
 find_package(Boost REQUIRED)
-find_package(Eigen REQUIRED)
```

Comment:
I'm surprised that omitting it didn't lead to any build errors.  I'll place them back in the CMakeList files.